guides/common/modules/proc_enabling-dependency-solving.adoc

```diff
@@ -0,0 +1,23 @@
+[id="enabling-dependency-solving_{context}"]
+= Enabling Dependency Solving
+
+You can enable dependency solving per Content View or by default for your whole {Project} environment.
+
+.Prerequisites
+
+* Dependency solving is useful only in limited contexts.
+Before enabling it, ensure you read and understand xref:Resolving_Package_Dependencies_content-management[]
+
+.Procedure
+
+To enable dependency solving per Content View in the {ProjectWebUI}
+
+. Select *Content > Content Views*.
+. In the list of content views, select the required content view.
+. Under *Details*, toggle *Solve dependencies*.
+
+To enable dependency solving by default:
```

Comment:
```suggestion
.Enabling dependency solving by default
```

Reply:
I'd have to change the first lead-in sentence in a similar way as well, and keep the word "procedure" because that word is an important part of the module template. So it would go like this:

`.Procedure to enable dependency solving per Content View`
`.Procedure to enable dependency solving by default`

What do you think? I must say I prefer the original version here, but if you think this ^ version works better, I can change it.

Reply:
Or did you mean I should split the procedure module into two? If so, then I think they should stay together because I expect them to also be (re)used together.

Also, there is a pretty important admonition (included as a prerequisite) that would have to be copied or used as a snippet in both procedures if I split this one.

Reply:
I'd say "split" because it's two different use cases and two different worflows: change the default value vs. change for one CV. Maybe use snippets to share text if possible?

Reply:
I agree with Max.

Reply:
Split into two separate modules.